composer.json

```diff
@@ -29,7 +29,9 @@
         "friendsofsymfony/jsrouting-bundle": "1.5.3",
         "willdurand/js-translation-bundle": "~2.2",
         "white-october/pagerfanta-bundle": "2.1.*@dev",
-        "gitonomy/gitlib": "~0.1"
+        "gitonomy/gitlib": "~0.1",
+        "gitonomy/git-bundle": "0.2.*",
+        "erusev/parsedown": "~1.5"
```

Comment:
This repo is not the best solution for markdown. Even if the idea is interessing, the one from michelf is still better (in terms of compatibility with markdown and issues). https://michelf.ca/projets/php-markdown/

On the other hand you should recommand the installation of php-ext-hoedown.